File: app/src/main/java/ir/mymessage/view/interfaces/LoginInterface.java
package ir.mymessage.view.interfaces;

import android.content.Context;

public interface LoginInterface {
    String getUserName();
    String getPassword();
    void setupLoginActivity();
    void onLoginClicked();
    void startDialogsActivity();
    void startSignupActivity();
    void showUsernamePasswordError();
    void showProgress();
    void hideProgress();
    Context getContext();
}

File: app/src/main/java/ir/mymessage/view/interfaces/LoginInterface.kt
package ir.mymessage.view.interfaces

import android.content.Context

interface LoginInterface {
    fun getUserName(): String
    fun getPassword(): String
    fun getContext(): Context
    fun setupLoginActivity()
    fun onLoginClicked()
    fun startDialogsActivity()
    fun startSignupActivity()
    fun showUsernamePasswordError()
    fun showProgress()
    fun hideProgress()
}
